apps/prairielearn/src/sync/fromDisk/assessments.js

```diff
@@ -268,6 +269,16 @@ function getParamsForAssessment(assessmentInfoFile, questionIds) {
   return assessmentParams;
 }
 
+// function extractInfo(qid) {
+//   const firstSlash = qid.indexOf('/');
+//   const sourceCourse = qid.substring(1, firstSlash);
+//   const questionDirectory = qid.substring(firstSlash + 1, qid.length);
+//   return {
+//     sourceCourse: sourceCourse,
+//     qid: questionDirectory
+//   }
+// }
+
```

Comment:
👀 what's this?

Reply:
This is code for splitting a shared question URL into the course identifier and question path. I was keeping it around because it was used in a past version of the code and I wasn't sure if I would need it again. 

I will remove it now.